scapy/scapy_pcp.py

```diff
@@ -258,7 +258,7 @@ class PCPRequest(Packet):
 class PCPResponse(Packet):
     name = "PCP Response"
     fields_desc = [ByteEnumField("version", PCPVersions.V2, PCP_VERSIONS),
-                   BitEnumField("R", PCPMessages.REQUEST, 1, PCP_MESSAGES),
+                   BitEnumField("R", PCPMessages.RESPONSE, 1, PCP_MESSAGES),
```

Comment:
This commit should be squashed with previous one. I see noreason to keep this in history.